src/cdk-experimental/config.bzl

```diff
@@ -0,0 +1,13 @@
+entryPointGroups = [
+    ["clipboard"],
```

Comment:
I'd prefer not to speculatively use the extra arrays here. I generally want to avoid "we might need this later" coding practices and focus on the current state.

Reply:
Fine by me. Though it already served the use of being visually more readable. Changed.